java/compaction/compaction-job-creation/src/test/java/sleeper/compaction/job/creation/CreateJobsTest.java

```diff
@@ -209,6 +209,69 @@ public void shouldCreateStandardCompactionsToConvertSplitFilesToWholeFiles() thr
         });
     }
 
+    @Test
+    void shouldCreateJobsWhenStrategyDoesNotCreateJobsForWholeFilesWithForceCreateJobsFlagSet() throws Exception {
```

Comment:
It took me a while to figure out why the strategy wouldn't create the jobs. For both of these tests, it'd be nice to be explicit that it's because there are fewer files than the batch size.

Reply:
Thanks, i've added comments describing why the strategy doesn't create jobs in these scenarios